index.html

```diff
@@ -229,21 +251,26 @@ <h3 class="list-group-item-heading">Git / Github</h3>
     <div class="row"> <!-- Begin first thumnail row -->
       <div class="col-sm-6">
         <div class="thumbnail"> <!-- Thumbnail 1 -->
-          <img src="..." alt="...">
+          <div class="embed-responsive embed-responsive-4by3">
```

Comment:
Should I get in the habit of naming bootstrap classes even if I think I'm only going to use it in one instance? (Ex: add .about-bowling-vid to the embed even if it's the only embed on the page)